forgerock-openbanking-uk-aspsp-common/src/main/java/com/forgerock/openbanking/common/model/data/FRCustomerInfo.java

```diff
@@ -46,7 +47,7 @@ public FRCustomerInfo(CustomerInfo customerInfo){
         this.givenName = customerInfo.getGivenName();
         this.email = customerInfo.getEmail();
         this.phoneNumber = customerInfo.getPhoneNumber();
-        this.birthdate = customerInfo.getBirthdate();
+        this.birthdate = customerInfo.getBirthdate().toDateTimeAtCurrentTime();
         this.address = new FRCustomerInfoAddress(customerInfo.getAddress());
```

Comment:
Changed birthdate from LocalDate to DateTime to avoid array representation when mapping to json to present the value on the UI.